sdk/cosmos/azure-cosmos/azure/cosmos/offer.py

```diff
@@ -24,13 +24,24 @@
 
 class ThroughputProperties(object):
     """Represents the throughput properties in an Azure Cosmos DB SQL API container.
-    To read and update throughput properties use the associated methods on the :class:`Container`.
+
+    To read and update throughput properties, use the associated methods on the :class:`Container`.
+    If configuring auto-scale, both `auto_scale_max_throughput` and `auto_scale_increment_percent` must be set and
+    the value of `offer_throughput` will not be allowed to be set in conjunction with auto-scale.
+
+    :keyword int offer_throughput: The provisioned throughput in request units per second as a number.
+    :keyword int auto_scale_max_throughput: The max auto-scale throughput. It should have a valid throughput
+     value between 1000 and 1000000 inclusive, in increments of 1000.
+    :keyword int auto_scale_increment_percent: is the % from the base selected RU it increases at a given time,
+     up to a maximum default value of 0. The increment percent should be greater than or equal to zero.
```

Comment:
this wording seems weird as well - what do you mean that the maximum is 0 if you then say that it should be greater than or equal to zero? did you mean to say minimum?

Reply:
yes I meant minimum thats my bad will change that now